src/ccl_cls.c

```diff
@@ -217,50 +292,31 @@ void ccl_angular_cls_limber(ccl_cosmology *cosmo,
         // Get integration limits
         get_k_interval(cosmo, trc1, trc2, l, &lkmin, &lkmax);
 
-        // Integrate
-        gslstatus = gsl_integration_qag(&F, lkmin, lkmax, 0,
```

Comment:
You got rid of the qag method. We should keep the old code the same and simply add a new method.

Reply:
this has just been moved into the `integ_cls_limber_quad` function.

Reply:
Ahhhh I missed it! Sorry! I'll soldier on!